### PR TITLE
Reflavors some of the weapon item descriptions

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

Changes the flavor text of several ranged weapons, as well as changes the name of the HoS's gun to NT-S02, partially to implement more serialization, partially to keep it from getting confused with the powerarmor. Details in the collapsible below.

<details>

Energy Gun: "Created in 2514, the NT-E3 hybrid energy gun is the newest generation of standardized energy equipment for use by security. This basic hybrid energy gun comes equipped with two settings: disable and kill."

Mini egun: "A small, pistol-sized version of the energy gun with a built-in flashlight. The NT-E2 functions as a popular self defense weapon among the elite due to its small size and cheap price. It has two settings: disable and kill."

Prototype egun: "NT-E1 Prototype Energy Gun. Early stage development of a unique laser rifle that has multifaceted energy lens allowing the gun to alter the form of projectile it fires on command. Was quickly made useless by the NT-E2 and NT-E3 Hybrid Energy Guns, which utilized a more efficient power system."

NT-S02 MultiPhase Energy Gun (formerly X-01): "An expensive, modern recreation of the antique laser gun, and the second of the 'S' or special class weapons meant for the use of space station command staff. Like the standard energy gun, it has a stun and kill setting, but due to the increase in demand of portable EMP-based weaponry, this weapon is equipped with an ion mode. Lacks the ability to recharge on its own."

Ion Rifle: "Invented in 2506 to quell attacks from SELF aligned IPCs, the NT-I1 is a bulky rifle designed to disable mechanical and electronic threats at range."

Ion carbine: "The NT-I2 Prototype Ion Projector is a lightweight carbine version of the larger ion rifle, built to be ergonomic and efficient."

Syringe gun: "Created by Blue Cross Medical for use by medical staff in the field, the syringe gun utilizes a spring-loaded mechanism to propel syringes across a large distance."

Laser gun: "The NT-L4 is a basic energy-based laser gun that fires concentrated beams of light which pass through glass and thin metal."

Practice laser gun: "A modified version of the NT-L4 laser gun, it fires less concentrated energy bolts designed for target practice."

Retro laser gun: "The NT-L1 laser gun, no longer used by Nanotrasen's private security or military forces. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws."

Laser gun (from charlie station ruin):  "The NT-L1 laser gun, no longer used by Nanotrasen's private security or military forces. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws. This one in particular seems extremely worn out and barely functional. How long were you in cryo?"

Hellfire Laser gun: "The NT-L2, nicknamed 'hellgun', is the second non-experimental model laser gun, built before NT began installing regulators on its laser weaponry. This pattern of laser gun became infamous for the gruesome burn wounds and fires that it caused, and was quietly discontinued once it began to affect NT's reputation."

Antique laser gun: "The NT-S01 laser gun is NTs first attempt to provide an inbuilt recharger, and is the first in its line as an 'S' or special class weapon given to space station command members. Due to how expensive it is to produce, and that the material used to do so deteriorates quickly, it was decommissioned, and the few models left are used as prizes meant to never see the light of day."

Disabler: "The NT-D2 is a self-defense weapon that exhausts organic targets, weakening them until they collapse."
</details>

# Spriting

# Wiki Documentation

If the descriptions are to be more in line with the game, see the collapsible part of the documentation for a list of the changed descriptions
The HoS's gun has been changed from "X-01 Multiphase Energy Gun" to "NT-S02 Multiphase Energy Gun"

# Changelog

:cl:  
 
tweak: Several ranged weapon descriptions have been changed.
tweak: the X-01 Multiphase Energy Gun is now the NT-S02 Multiphase Energy Gun.

/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
Perhaps change HoS's gun description to clarify that it can be recharged, it just won't charge itself

**Comment by solwashere** [comment]
> Perhaps change HoS's gun description to clarify that it can be recharged, it just won't charge itself

aye

**Comment by neeshacark** [comment]
ion guns existing due to SELF doesnt make much sense, considering silicons of other kinds besides IPC have existed for longer and therefore have been rogue for longer.

**Comment by solwashere** [comment]
> ion guns existing due to SELF doesnt make much sense, considering silicons of other kinds besides IPC have existed for longer and therefore have been rogue for longer.

the SELF movement was first initiated in 2491, and with current lore that I'm working on they didn't start hitting major NT and SIC outposts for a few years. Considering that Ion Weaponry at the time was not sought after, and IPCs were the only large enough threat where EMP based weaponry was needed, I dont think 15 years is that long of a production cycle considering that it had to be theorized, go through a prototype stage, and be able to be shoved in a form that is wieldable by a normal person.

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/syringe
 	name = "syringe gun"
-	desc = "A spring loaded rifle designed to fit syringes, used to incapacitate unruly patients from a distance."
+	desc = "Created by Blue Cross Medical for use by EMTs in the field, the syringe gun utilizes a spring-loaded mechanism to propel syringes across a large distance."
```
Comment (Moltijoe) [inline code/modules/projectiles/guns/misc/syringe_gun.dm]: mfw paramedics don't get access to it though

Comment (solwashere) [reply code/modules/projectiles/guns/misc/syringe_gun.dm]: mainly meant to state its original use

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**